### PR TITLE
Create req.txt

### LINKED ISSUE #10: Shared `X25519_PRIVATE_KEY` for Development
Can somebody work on generating a new `X25519_PRIVATE_KEY` for the ubiquibot organization? We need to share it for development purposes.

1. Generate a new key
2. Encrypt a new `evmPrivateKeyEncrypted` (no funds!) and add to the org bot config
3. Add the shared test key to the `comment-incentives` readme.

### PR DESCRIPTION
**Pull request by EresDev**
Resolves #10 

The new evmPrivateKeyEncrypted generated for address 0x3a2E44e10AbEf5CB4a6E492c5ba93d30068d2D95 (no funds).

### CODE REVIEW

**Comment by web4er** [comment]
Need to document a private key too

**Comment by web4er** [comment]
I am starting a new repository for a new project.

**Comment by EresDev** [comment]
### things to know
- the PR introduces a new file
- the new fils will contain one or more keys to be used for development
- all keys should not contain funds

File: req.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+request for key
```
Comment (web4er) [inline req.txt:1]: You should add one working key.

Comment (EresDev) [reply req.txt:1]: Ok, I will add some in a separate PR.

**Review by web4er — CHANGES_REQUESTED** [review]
An automated script to generate key would help

**Pull request merged**